src/app/core/header/header.component.html

```diff
@@ -1,4 +1,10 @@
 <header class="container d-flex my-2 justify-content-between">
   <h1>Courses</h1>
-  <courses-login></courses-login>
+  <a routerLink="/login"
+     class="btn btn-primary"
+     *ngIf="loginShown">Log in</a>
```

Comment:
Please consider moving of text and closing tag to the next lines - it'll be easier to read

Reply:
Will create a separate PR for this